code/modules/mob/living/say.dm

```diff
@@ -126,6 +128,9 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	else if(stat == UNCONSCIOUS)
 		if(!(unconscious_allowed_modes[message_mode]))
 			return
+	else if(is_paralyzed)
```

Comment:
```suggestion
	else if(IsParalyzed())
```

Reply:
Fixed that, and also changed the syntax that I had used from the line above to reflect the same